watershed_workflow/utils.py

```diff
@@ -16,7 +16,7 @@
 import shapely.affinity
 import rasterio
 
-import watershed_workflow
+import watershed_workflow 
```

Comment:
This import probably shouldn't be here -- importing at the top level is a bad thing.  It looks like this should instead be:

`import watershed_workflow.crs`

Can you fix this while you're at it please